docs/db.md

```diff
@@ -2,53 +2,59 @@
 
 Our database is postgresql@14.7 with prisma to manage the schema and migrations. Each package can have its own prisma schema if it has its own database. At this moment, only the api project has a database.
 
-[Database schema](./../packages/db/prisma/schema.prisma)
-
+[Database schema](./../packages/db/src/schema.prisma)
 ## Setup
 
-1. Install postgresql version 14.7
-1. Create database for the project
-1. Set the `DATABASE_URL` env var in `packages/api/.env.local` and `packages/db/.env.local` to the postgres connection string. You may need to include a username and password in the connection string.
-1. Run `nx prisma db migrate reset` to scaffold the database schema.
-1. Run `nx seed db` to insert static verse data into the database.
+1. [Install](https://www.postgresql.org/download/) postgresql version 14 (latest version).
+1. Create database for the project. You can perform this task using the included `pgAdmin` tool.
+1. Create a copy of file: [packages/api/.env](./../packages/api/.env) in same directory and rename it to: `.env.local`.
+1. Create new file: `packages/db/.env.local`.
```

Comment:
You don't need to copy the .env file, you can just create `.env.local` in both files. Our build system loads both files. `.env.local` will override what is in `.env`. That's why we only need the DATABASE_URL in that file because it is unique to each machine.

Let's combine these two steps to say something like `Create a .env.local file in the api and db projects for local environment variables.`